README.md

```diff
@@ -7,6 +7,26 @@
 Plug 'mcchrish/nnn.vim'
 ```
 
+### Usage
+
+To open nnn as a file picker in vim/neovim, use the command `:NnnPicker` or
+`:Np` or the key-binding `<leader>n`. Once you select one or more files and quit
+nnn, vim/neovim will open the first selected file and add the remaining files to
+the arg list/buffer list. If no file is explicitly selected, the last selected
+entry is picked.
+
+#### Custom mappings
```

Comment:
Please add the following in this section:

Create a function/mapping to open the _directory_ of currently open file in `nnn`:

    nnoremap <leader>nd :call NnnPicker('%:p:h')<CR>

Reply:
What does the `:p` do BTW?

Reply:
@jarun it expands to full path. e.g. `/Users/mcchris/plugin/nnn.vim` instead of just `plugin/nnn.vim`

Reply:
Thanks for explaining! I am no good at vim plugins.

Reply:
Add some documentation.